### PR TITLE
Add a way to specify files to ignore

### PR DESCRIPTION
**Pull request by thiblahute**
This way we can iteratively pylint files in a big project ensuring that no pylint regression happens meanwhile.

### CODE REVIEW

**Comment by thiblahute** [comment]
Build failure does not seem to be related to that commit at all tbh.

**Comment by thiblahute** [comment]
Hello there,

Is that project maintained or should I use a fork? I really need that commit so we can start using linting incrementally :)

File: git_pylint_commit_hook/commit_hook.py [hunk 1]
```diff
@@ -87,7 +95,7 @@ def _parse_score(pylint_output):
 
 def check_repo(
         limit, pylint='pylint', pylintrc='.pylintrc', pylint_params=None,
-        suppress_report=False):
+        suppress_report=False, ignored_files=[]):
```
Comment (sandipagarwal) [inline git_pylint_commit_hook/commit_hook.py]: Function arguments' default value should not be set to a list, it being mutable.

**Comment by sebdah** [comment]
Manually merged and now released in 2.2.0.

**Pull request closed without merging**